package.json

```diff
@@ -57,7 +61,21 @@
         "node": ">=4.2.1"
     },
     "scripts": {
-        "test": "node tests/all-tests.js",
+        "pretest": "npm run pretest:clean && npm run pretest:instrument && npm run pretest:copyUninstrumentedCode",
+        "pretest:clean": "node node_modules/rimraf/bin.js reports/* coverage/* instrumented/*",
+        "pretest:instrument": "npm run pretest:instrumentNodeModules && npm run pretest:instrumentConfigs",
+        "pretest:instrumentNodeModules": "node node_modules/istanbul/lib/cli.js instrument --complete-copy -o instrumented/universal/gpii/node_modules gpii/node_modules -x '**/lib/**' -x 'testing/**' -x 'tests/**'",
+        "pretest:instrumentConfigs": "node node_modules/istanbul/lib/cli.js instrument --complete-copy -o instrumented/universal/gpii/configs gpii/configs",
+        "pretest:copyUninstrumentedCode": "node scripts/copyUninstrumentedCode.js",
+        "test": "npm run test:node && npm run test:browser",
+        "test:browser": "npm run pretest:instrument && npm run pretest:copyUninstrumentedCode && node node_modules/testem/testem.js ci --file tests/testem.js --skip PhantomJS",
```

Comment:
At line 71, 72 that define commands for running browser and node tests individually, without running `pretest:clean` as a initial step, could the left over reports from the last run affect the output report?